tests/kani/Intrinsics/PtrGuaranteedCmp/eq.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-verify-fail
```

Comment:
Is `kani-verify-fail` required?

Reply:
It was not needed, but I changed the tests to have two positive harnesses instead.